src/App.tsx

```diff
@@ -281,6 +287,13 @@ const Screens = (t: TFunction) => {
         title: t('viewNames:positiveResult'),
         ...standardProps
       }
+    },
+    {
+      name: ScreenNames.AgeCheck,
+      component: AgeCheck,
+      options: {
+        ...standardProps
```

Comment:
Need to disable header as an option here, check the React Navigation stack navigator docs